### ISSUE TITLE
Number of- and position of items in a WPF ComboBox are not announced

### ISSUE THREAD

**Issue by mjcheetham** [nvaccess/nvda#8187]
### Steps to reproduce:
1. Create a new blank WPF application from Visual Studio (File > New > Project > C# > WPF App)
2. Add a simple `<ComboBox/>` control with a new few items:
```xaml
<Window x:Class="WpfApp1.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        Title="MainWindow" Height="450" Width="800">
    <Grid>
        <ComboBox>
            <ComboBoxItem>Foo</ComboBoxItem>
            <ComboBoxItem>Bar</ComboBoxItem>
            <ComboBoxItem>Fizz</ComboBoxItem>
            <ComboBoxItem>Buzz</ComboBoxItem>
            <ComboBoxItem>Baz</ComboBoxItem>
        </ComboBox>
    </Grid>
</Window>
```
3. Build and run the application (Debug > Start Debugging, or <kbd>F5</kbd>)
4. Start NVDA
5. Bring the test application to the foreground and tab to focus on the combo box.
6. Press <kbd>Alt</kbd>+<kbd>Down</kbd> to expand the combo box
7. Use the arrow keys to move through the combo box items

### Expected behaviour:
NVDA should announce the position and total number of items in the menu. For example:
> "Fizz, not selected, 3 of 5"

### Actual behaviour:
NVDA only announces the text of the menu item, giving no indication of the position in the menu, or the total number of options available.

![image](https://user-images.githubusercontent.com/5658207/38862181-31212abc-422c-11e8-91da-625eec3ab549.png)

### System configuration:
**NVDA version:**
2018.1.1

**NVDA Installed or portable:**
Installed

**Other information:**
Windows Narrator correctly announces the position and total number of elements in the list. I.e.,
> "Fizz, not selected, 3 of 5"

**Windows version:**
Windows 10 Version 1709 Build 16299.371

**Name and version of other software in use when reproducing the issue:**
None

### Other questions:

**Does the issue still occur after restarting your PC?**
Yes

**Have you tried any other versions of NVDA?**
No

**Comment by Adriani90** [comment]
cc: @feerrenrut, @leonardder

**Comment by DataTriny** [comment]
Hi, I can reproduce the issue in NVDA 2019.1.1 on Windows 10 Education, version 1809. Note that this is not a WPF specific behavior as I got the exact same results inside a Windows Forms application.

**Comment by DataTriny** [comment]
I just realized that this is not even specific to .NET Framework UI controls since I get the same behavior on NVDA's GUI ComboBoxes like in the Synthesizer settings tab for example. Is this intensional?

**Comment by feerrenrut** [comment]
It's worth questioning whether the extra verbosity of this outweighs the utility. Another option would be to announce the number of options in the combo box when first expanding it.

**Comment by Adriani90** [comment]
I don't know if this would impact performance, especially in combo boxes with many items. Would it be an alternative to implement an unassigned gesture for counting the items and reporting the position and total number of items on key press? At least from my point of view I don't really need this information if I know that items are in an alphabetical order or so. A sighted person does not have this information either.

**Comment by DataTriny** [comment]
It's true @Adriani90 I don't think this information is very useful either.
But, I think there is a problem when using only a Braille display: there is nothing to tell the user that he/she is at the beginning or the end of the list. When navigating with speech, you can deduce this because nothing is spoken when pressing up or down arrow.

**Comment by Adriani90** [comment]
I agree. There should be a patern which shows in braille the top or bottom of list (i.e. by showing a line of dashes for two seconds or so. If possible, for the top of list there would be a line of c (dots 14) and for the bottom a line of dots 78.

**Comment by feerrenrut** [comment]
@Adriani90 @DataTriny That's true, but would be better to be discussed on a separate issue so as not to derail this one. I'm going to mark these comments as off-topic, please could one of you create a separate issue for Braille users to have a better indication of the start or end of a combo box? Thanks!
